src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -18,10 +18,9 @@ Rename "fluid.prefs.enactors" to "fluid.prefs.enactor"
 
 <div class="infusion-docs-note"><strong>Note:</strong> According to the [comment](https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/FluidView.js#L38-L39) on the implementation for relay components, in Infusion 2.0, relay components will be renamed back to its original names. If the rename has been made, this section can be ignored.</div>
```

Comment:
Can we update this comment to clearly describe the reality that the base component grades have been renamed?

Also, can we document other 1.5-2.0 changes such as the removal of demands blocks, non-dynamic invokers, etc.?